packages/thirdweb/src/react/web/wallets/shared/ConnectWalletSocialOptions.tsx

```diff
@@ -87,27 +90,35 @@ export const ConnectWalletSocialOptions = (
     apple: locale.signInWithApple,
   };
 
-  const authOptions = defaultAuthOptions; // TODO: Fetch these from the backend
+  const { data: authOptions, isLoading } = useQuery({
+    queryKey: ["auth-options", wallet.id],
+    queryFn: async () => {
+      if (isEcosystemWallet(wallet)) {
+        return (
+          (await getEcosystemWalletAuthOptions(wallet.id)) ?? defaultAuthOptions
+        );
+      }
+      return wallet.getConfig()?.auth?.options ?? defaultAuthOptions;
+    },
+    staleTime: 0,
```

Comment:
this is the default, if you meant to fetch this once and never again (per session) use: `Infinity` instead